src/functions-templates/javascript/edge/image-external/{{name}}.js

```diff
@@ -0,0 +1,9 @@
+// Let's serve an image of a kitten from the internet
+
+export default async () => {
+  // fetch() is supported natively by Deno!
+  // Returning the awaited response automatically sets the
+  // content-type headers!
+  const kitten = await fetch('https://placekitten.com/g/300/300')
+  return kitten
+}
```

Comment:
As above, I think it would be really beneficial to match the examples to the example site and not use arrow functions. I won't add any further comments on this, but they apply to all files.

Reply:
This one matches: https://edge-functions-examples.netlify.app/example/image -- I guess I was looking at an older version of it though, which split the internal and external image retrieval. I can update it to match, but then eslint complains about unreachable code (since there are two return statements). It might also not be necessary to include, since there are plenty of other templates to choose from. Any strong feelings on this one @whitep4nth3r?